src/js/step.js

```diff
@@ -287,7 +287,7 @@ export class Step extends Evented {
     if (isFunction(this.options.scrollToHandler)) {
       this.options.scrollToHandler(element);
     } else if (
-      isElement(element) &&
+      element instanceof Element &&
```

Comment:
Perhaps we should change `isElement` to check for `Element` instead?

Reply:
`isElement `is used elsewhere and is not documented this way. It checks only `HTMLElement`.

Especially, it will create a bug in: https://github.com/shipshapecode/shepherd/blob/4885a3e7c9668005d71d0de70296f553b6063935/src/js/tour.js#L298-L299

Because `focus` is a method of `HTMLElement`, not `Element`.

If you really want to implement it in type-check.js, then I suggest :
 - Either rename `isElement` -> `isHTMLElement`; and implement an `isElement` that checks for `Element`
 - Or implement an `isAnyElement` that checks for `Element`

Reply:
Let's rename `isElement` to `isHTMLElement` and add an `isElement` that checks for `Element` please.

Reply:
I'm sorry, I won't have time to make this refactoring, would you like to do it?